crt_portal/cts_forms/templates/forms/report_class.html

```diff
@@ -0,0 +1,59 @@
+{% extends "forms/report_base.html" %}
+{% load static %}
+{% block form_questions %}
+<fieldset class="usa-fieldset" aria-label="{{ current_step_name }}">
+  <legend class="usa-sr-only">{{ current_step_name }}</legend hidden>
+  {% for field in wizard.form.visible_fields %}
+      {% if field.label == 'Other' %}
+        <div class="usa-prose margin-bottom-2 padding-left-3" id='other-class-option' >
+      {% else %}
+        <div class="usa-prose margin-bottom-2">
+        <label for="{{ field.id_for_label }}" class="big margin-bottom-0">{{ field.label }}</div>
+      {% endif %}
+      {% if field.errors %}
+        <div class="usa-alert usa-alert--warning usa-alert--slim">
+          <div class="usa-alert__body">
+          <em class="usa-alert__text">{{ field.errors }}</em>
+          </div>
+        </div>
+      {% endif %}
+    <div class="margin-bottom-1">
+    {% if field.label == 'Other' %}
+        <div>
+          <p><em><label for="{{ field.id_for_label }}">{{ field.help_text }}</label></em></p>
+          {{ field }}
+          <em id="word_count_area" hidden>
+            <span id="display_count">10</span>
+            <span id="count_message">word(s) remaining</span>
+          </em>
+          <div id="word-limit-alert" class="usa-alert usa-alert--warning usa-alert--slim" hidden>
+            <div class="usa-alert__body">
+              <em class="usa-alert__text">10 word limit reached</em>
+            </div>
+        </div>
+      </div>
+    {% else %}
+      <p><em>{{ field.help_text }}</em></p>
+      {{ field }}
+    {% endif %}
+    </div>
+  {% endfor %}
+
+</fieldset>
+<div class="margin-bottom-5"></div>
+{% endblock %}
+
+{% block page_js %}
+<script>
+var otherbox = document.getElementById("other-class-option");
+otherbox.setAttribute("hidden","");
+var otherCheckLabel = document.getElementsByTagName("LABEL").item(14);
```

Comment:
I think selecting this element based on ID rather than on ordering would make it less fragile; selecting using ordering means that this JS will break if new protected classes are added or removed.

Reply:
I'm also hoping we can avoid in-line JS in favor of JS served up via assets; if we have JS running both inline and through the assets it will be trickier to debug as opposed to having all the JS in one place.

Reply:
The ids change depending on the PK of the data, so that isn't going to be consistent across environments. The order has to be consistent, as described by the requirements so that is really more stable in this case. 

I am setting the variables in the template because they have to be dynamically generated.